a3p-integration/proposals/a:upgrade-next/package.json

```diff
@@ -4,6 +4,10 @@
     "sdkImageTag": "unreleased",
     "planName": "UNRELEASED_UPGRADE",
     "upgradeInfo": {
+      "binaries": {
+        "any": "https://github.com/Agoric/agoric-sdk/archive/6982dbb493e87f782ad67fdbd98ab93386d56fc0.zip//agoric-sdk-6982dbb493e87f782ad67fdbd98ab93386d56fc0?checksum=sha256:767f6444d92cec394eb7cb675af56eddd95fc1886e7a7c7775dec6141c55738d"
+      },
+      "source": "https://github.com/Agoric/agoric-sdk/archive/6982dbb493e87f782ad67fdbd98ab93386d56fc0.zip?checksum=sha256:767f6444d92cec394eb7cb675af56eddd95fc1886e7a7c7775dec6141c55738d",
```

Comment:
@gibson042 and I were talking through this and we believe that agoric-sdk should ask synthetic-chain to disable validation of the upgrade info, as it's impossible to have a correct value in here that's not a moving target. This will require a new option to be plumbed into synthetic-chain to use `--no-validate` in `run_prepare.sh`

Reply:
Got it. Restoring assignment of issue to @gibson042

Reply:
I think this is blocked on https://github.com/Agoric/agoric-3-proposals/issues/146 .

Reply:
This PR now uses https://github.com/Agoric/agoric-3-proposals/pull/148